docs/index.rst

```diff
@@ -39,20 +39,22 @@ Publicly available data access and web services are currently in development.
    connect/nci_basics.rst
    connect/jupyter.rst
 
+
 .. toctree::
    :maxdepth: 1
    :caption: User Guide
 
-   query/Getting Started.ipynb
-   query/List Products.ipynb
-   query/Loading Data.ipynb
-   query/Advanced Querying.ipynb
-
-.. toctree::
-   :maxdepth: 1
-   :caption: Notebook Gallery
+   notebooks/Getting_started/README.rst
+   notebooks/DEA_datasets/README.rst
+   notebooks/Integrating_external_data/README.rst
+   notebooks/Index_calculation/README.rst
+   notebooks/Temporal_analysis/README.rst
+   notebooks/Composite_generation/README.rst
+   notebooks/Image_classification/README.rst
+   notebooks/Outputting_data/README.rst
+   notebooks/Workflows/README.rst
+   notebooks/Scripts/README.rst
 
-   notebooks/index
```

Comment:
I'm uncomfortable with including files so deep into a different repo, but I couldn't find a way to get it to work with toctree.

Reply:
I was trying to do a:
```rst
.. include:: notebooks/index.rst
```
But it doesn't work nicely due to relative paths.

I also tried doing
```rst
.. toctree::
   :glob:
   :caption: 

   notebooks/*/README.rst
```
But this will include the sections out of order. A possible option would be to rename the folders in the `dea-notebooks` directory.

Thoughts?